drafts/dec-14.md

```diff
@@ -0,0 +1,9 @@
+Here are some of the highlights for the [December 14](https://github.com/ipfs/pm/issues/76) Sprint:
+
+### Releases
+* @whyrusleeping shipped [fs-repo-migrations](https://github.com/ipfs/fs-repo-migrations)! These are migrations for the filesystem repository of ipfs clients. This tool is written in Go, and developed alongside go-ipfs, but it should work with any repo conforming to the fs-repo specs.
+* @Dignifiedquire did some great work with [rust-multiaddr](https://github.com/Dignifiedquire/rust-multiaddr), which is a Rust implementation of @jbenet's [multiaddr](https://github.com/jbenet/multiaddr).
+### Updates
```

Comment:
Missing line break before this header, so it doesn't format properly in markdown